backend/legacyapi/task.go

```diff
@@ -333,8 +334,8 @@ type TaskPatch struct {
 	Statement         *string `jsonapi:"attr,statement"`
 	SchemaVersion     *string
 	RollbackEnabled   *bool `jsonapi:"attr,rollbackEnabled"`
-	RollbackStatement *string
-	RollbackError     *string
+	RollbackStatement *sql.NullString
```

Comment:
Why do we need sql.NullString? This is PATCH, empty string sets an empty string. Nil will not set things.

Reply:
Because we want to patch this json field to `null`.

RollbackStatement=nil => do nothing
RollbackStatement=&sql.NullString{Valid:false} => patch to null
RollbackStatement=&sql.NullString{Valid:true, Value:"hi"} => patch to "hi"

Reply:
Or just use a RollbackDone bool to tell if it's finished...

Reply:
I agree that using nil vs "" might be too flaky. If we do want more details we may use a new bool field.